locales/fr/translation.json

```diff
@@ -676,6 +676,7 @@
   "Within this basal segment, we are omitting a suspend event that didn't end. This may have resulted from changing the date & time settings on the device or switching to a new device. As a result, this basal segment may be inaccurate.": "Dans ce débit de base, nous avons omis de vous afficher un évenement car il ne s'est jamais terminé. Ceci peut être dû à un changement de date ou d'heure sur votre système ou au changement pour un nouveau système. Par conséquent ce débit de base peut être imprécis.",
   "Would you like to set up data storage for someone’s diabetes data?": "Souhaitez-vous paramétrer le stockage des données pour une personne avec un diabète ?",
   "Wrong username or password.": "Erreur d'identifiant ou de mot de passe.",
+  "Wrong username or password. Your account may have been lock for {{delayBeforeNextLoginAttempt}} minutes after {{maxFailedLogin}} failed attempts.": "Erreur d'identifiant ou de mot de passe. Votre compte à peut être été vérouillé perndant {{delayBeforeNextLoginAttempt}} minutes après {{maxFailedLogin}} échecs",
```

Comment:
same as above
```suggestion
  "Wrong username or password. For your security your account has been locked for {{delayBeforeNextLoginAttempt}} minutes because you have reached the maximum number of logon attempts.": "Erreur d'identifiant ou de mot de passe. Pour votre sécurité, votre compte a  été temporairement vérouillé pendant {{delayBeforeNextLoginAttempt}} minutes car vous avez dépassé le nombre d'échecs maximum",
```